tests/modules/eventloop.c

```diff
@@ -130,12 +130,12 @@ int sanity(ValkeyModuleCtx *ctx, ValkeyModuleString **argv, int argc) {
     }
     if (ValkeyModule_EventLoopAdd(-1, VALKEYMODULE_EVENTLOOP_READABLE, onReadable, NULL)
         == VALKEYMODULE_OK || errno != ERANGE) {
-        ValkeyModule_ReplyWithError(ctx, "ERR out of range fd should fail");
+        ValkeyModule_ReplyWithError(ctx, "ERR out-of-range fd should fail");
```

Comment:
This is a user error message, so we should avoid changing it in case anyone has taken a character-by-character dependency on it.